core/services/feeds/models.go

```diff
@@ -118,6 +118,7 @@ type FeedsManager struct {
 	IsConnectionActive bool
 	CreatedAt          time.Time
 	UpdatedAt          time.Time
+	DeletedAt          *time.Time
```

Comment:
I think that it might be better to call this `DisabledAt`
```suggestion
	DisabledAt          *time.Time
```